package.json

```diff
@@ -59,14 +59,15 @@
   "types": "./index.d.ts",
   "scripts": {
     "lint": "populist lib/*.js lib/winston/*.js lib/winston/**/*.js",
-    "pretest": "npm run lint",
-    "test": "nyc --reporter=text --reporter lcov npm run test:mocha",
-    "test:mocha": "mocha test/*.test.js test/**/*.test.js --exit",
+    "test": "mocha './test/**/*.test.js' --recursive --exit",
```

Comment:
Use Mocha's (config file](https://mochajs.org/#configuring-mocha-nodejs) instead (`.mocha.yml`). Also the added path is duplicating the `recursive` functionality.

Reply:
I'll move the `--recursive` & `--exit` cli flags out to a root configuration; I'm not aware of anything else that I would define in the configuration at this point though.

>Also the added path is duplicating the recursive functionality.
Can you elaborate on this?